libs/designsystem/src/lib/components/list/directives/infinite-scroll.directive.ts

```diff
@@ -1,16 +1,16 @@
 import {
-  Directive,
   AfterViewInit,
-  OnDestroy,
-  HostListener,
+  Directive,
+  ElementRef,
   EventEmitter,
+  HostListener,
   Input,
-  Output,
-  ElementRef,
   NgZone,
+  OnDestroy,
+  Output,
 } from '@angular/core';
-import { Subject, fromEvent } from 'rxjs';
-import { debounceTime, takeUntil, filter, map } from 'rxjs/operators';
+import { fromEvent, Subject } from 'rxjs';
+import { debounceTime, filter, map, takeUntil, tap } from 'rxjs/operators';
```

Comment:
Remove unused import
```suggestion
import { debounceTime, filter, map, takeUntil } from 'rxjs/operators';
```

Reply:
Should be resolved by 33362be